### DISCUSSION TITLE
Self hosting

### DISCUSSION THREAD

**Discussion by ghost** [Budibase/budibase#5452] [Help]
Hey there everyone 
I am not able to log into my budibase after fresh docker and budibase cli setup 
I shows "logged in successfully"
But nothing happens at all kindly help me in resolving thus issue

**Comment by melohagan** [comment]
Hi @muhammadAazim 

Would you be able to provide any errors from the console / network tabs when you log in please? ( Fn+F12 on Chrome ).

**Comment by ghost** [comment]
![IMG_20220419_194533](https://user-images.githubusercontent.com/82491981/164024592-9ad948e8-928c-4ed4-8c13-a7657b1e3dde.jpg)

**Reply by melohagan** [reply]
And if you open up the network tab of the browser window, do you see any red errors in there?

You could also try a restarting the docker services, or even remove them and re-install. 
If that fails, the docker logs for the bbworker service may be helpful to see as well.
